apps/ledger-live-mobile/src/screens/PTX/Earn/EarnInfoDrawer.tsx

```diff
@@ -19,31 +22,50 @@ export function EarnInfoDrawer() {
     await dispatch(setEarnInfoModal({}));
     await setModalOpened(false);
   }, [dispatch]);
-  const { message, messageTitle } = useSelector(earnInfoModalSelector);
+  const { message, messageTitle, learnMoreLink } = useSelector(earnInfoModalSelector);
 
   useEffect(() => {
     if (!modalOpened && (message || messageTitle)) {
       openModal();
     }
   }, [openModal, message, messageTitle, modalOpened]);
 
+  const onLearMorePress = useCallback(() => {
+    if (learnMoreLink) {
+      Linking.openURL(learnMoreLink);
+    }
+  }, [learnMoreLink]);
+
+  const { colors } = useTheme();
+
   return (
     <QueuedDrawer isRequestingToBeOpened={modalOpened} onClose={closeModal}>
-      <Flex rowGap={52}>
+      <Flex rowGap={32}>
         <Track onMount event="Earn Info Modal" />
-        <Flex rowGap={56}>
-          <Flex rowGap={16}>
-            <Text variant="h4" fontFamily="Inter" textAlign="center" fontWeight="bold">
-              {messageTitle}
-            </Text>
-            <Text variant="body" lineHeight="21px" color="neutral.c70" textAlign="center">
-              {message}
-            </Text>
-          </Flex>
+        <Flex rowGap={16} alignItems="center">
+          <Circle size={64} bg={colors.lightGrey}>
+            <Svg width={32} height={32} viewBox="0 0 33 34">
+              <Path
+                fill={colors.darkGrey}
+                d="M0.399902 16.9999C0.399902 8.09167 7.60811 0.899902 16.4999 0.899902C25.3924 0.899902 32.5999 8.10739 32.5999 16.9999C32.5999 25.8924 25.3924 33.0999 16.4999 33.0999C7.59167 33.0999 0.399902 25.8917 0.399902 16.9999ZM16.4836 9.23325L16.4751 9.23327L16.4669 9.23325C16.436 9.23325 16.4054 9.23452 16.3752 9.23702C15.5854 9.29214 14.9669 9.94551 14.9669 10.7499C14.9669 11.5455 15.6145 12.2666 16.4836 12.2666C16.9294 12.2666 17.3024 12.0659 17.551 11.8173C17.7995 11.5688 18.0002 11.1958 18.0002 10.7499C18.0002 9.90079 17.3275 9.29911 16.5899 9.23832C16.5549 9.23497 16.5194 9.23325 16.4836 9.23325ZM17.6002 16.9999C17.6002 16.3924 17.1077 15.8999 16.5002 15.8999C15.8927 15.8999 15.4002 16.3924 15.4002 16.9999V25.3333C15.4002 25.9408 15.8927 26.4333 16.5002 26.4333C17.1077 26.4333 17.6002 25.9408 17.6002 25.3333V16.9999Z"
```

Comment:
I assume that this is the "info" icon
It should already exist in the native-ui lib
Probably this one: https://github.com/LedgerHQ/ledger-live/blob/develop/libs/ui/packages/native/src/icons/Info.tsx

If it's not there, it would be better if it is added there and imported instead of written directly in the body of the component

Reply:
I had considered extracting it into that package. However, a different info icon exists there already, and I cannot seldom make the decision of replacing one with the other; adding it under a different name would serve as a potential source of confusion. Would naming it as a component and linking it to this discussion suffice for now?

Another thing that I could do is to extract it to a local module under `ledger-live-mobile`.

Reply:
There's one here called `InfoAltFill` that looks the same:  https://react-ui-storybook.vercel.app/?path=/story/asorted-icons-icons-legacy--single-icon&args=name:InfoAltFill&globals=measureEnabled:true;outline:true

You might even be able to import this one from @ledgerhq/ui-shared, since it's just an Svg (and not platform specific). If not, and this icon variant is missing from the mobile lib, it is definitely worth adding.